tools/docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
     image: ${DEV_DOCKER_TAG_BASE}/awx_devel:${TAG}
     container_name: tools_awx_1
     hostname: awx
-    command: /start_development.sh
+    command: /usr/bin/launch_awx.sh
```

Comment:
Since `/usr/bin` is in the PATH, this can just be `launch_awx.sh`

Reply:
ahh, good point.